src/clr-angular/utils/_theme.dark.clarity.scss

```diff
@@ -454,6 +454,10 @@ $clr-forms-radio-disabled-background-color: hsl(0, 0%, 0%);
 $clr-forms-radio-disabled-mark-color: $clr-forms-checkbox-disabled-mark-color;
 $clr-forms-radio-disabled-shadow: $clr-forms-checkbox-checked-shadow;
 
+// Range
+$clr-forms-range-progress-fill-color: hsl(198, 65%, 57%);
```

Comment:
These should not be hardcoded HSL values. Get colors from `clr-angular/color/_variables.color.scss`

Reply:
I'm using the colors from the progress bar and there the colors are hardcoded. In `clr-angular/color/_variables.color.scss` is no matching color.
Should I add them there and then use the variables?

Reply:
You may want to wait for https://github.com/vmware/clarity/pull/3892 to get merged in then...

Reply:
Now that #3892 is merged we should try to get this PR updated to use the custom prop variables if possible.

Reply:
Are these colors available as variables now?

Reply:
![Screenshot from 2019-10-18 17-06-51](https://user-images.githubusercontent.com/53469024/67101145-c35ec800-f1c9-11e9-897c-06498361a763.png)
No, those are still hardcoded.

Reply:
Alright. The dark theme is a bit of an oddity right now in either case. Things will be much better once custom properties becomes _the_ way to do themes.